app/views/welcome/_footer.html.erb

```diff
@@ -0,0 +1,25 @@
+  <footer class="page-footer grey darken-4">
```

Comment:
Always check for the indentation, it should be consistent 🤓, please indent this file correctly.